server.js

```diff
@@ -5322,6 +5322,63 @@ camp.route(/^\/vscode-marketplace\/(d|v|r)\/(.*)\.(svg|png|gif|jpg|json)$/,
   })
 );
 
+// Eclipse Marketplace integration.
+camp.route(/^\/eclipse-marketplace\/(d|dm|v|favorited)\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var type = match[1];
+  var project = match[2];
+  var format = match[3];
+  var apiUrl = 'https://marketplace.eclipse.org/content/' + project + '/api/p';
+  var badgeData = getBadgeData('', data);
```

Comment:
Best to put `'eclipse marketplace'` in there for benefit of the error badges. Will also save you a line of code in the version badge.